lib/mayaUsd/ufe/UsdAttribute.cpp

```diff
@@ -118,7 +117,11 @@ getUsdAttributeValueAsString(const PXR_NS::UsdAttribute& attr, const PXR_NS::Usd
         return os.str();
     }
 
-    UFE_ASSERT_MSG(false, kErrorMsgFailedConvertToString);
+    try {
+        UFE_ASSERT_MSG(false, kErrorMsgFailedConvertToString);
```

Comment:
Because this asserts on false, it will always assert --- and you never want it to assert.  Just remove the line and return an empty string.

Reply:
I actually left this one here on purpose. Asserting here would mean that for some reason we were unable to get the attribute value as a string. So I wanted this to fire so that we would see it and possibly make a fix.